project-p2-layouts-master/code/style.css

```diff
@@ -0,0 +1,252 @@
+}
+
+p {color: rgb(108, 164, 140);
+  font-family: 'Roboto Slab', serif;
+  font-size: 30px;
+  text-align: center;
+  margin-bottom: 5px;
+  margin-top: 5px; 
+  padding-top: 2px;
+  padding-left: 300px;
+  padding-right: 250px;
+  padding-bottom: 20px;
+}
+
+.fa {
+padding: 10px;
+width: 20px;
+height: 20px;
+text-align: center;
+margin: 5px
+}
+
+.fa:hover {
+opacity: 0.5;
+}
+
+.fa-instagram {
+background:rgb(251, 221, 218);
+color:rgb(249, 250, 249);
+border-radius:50%; 
+font-size: 20px; 
+font-weight: bolder;
+line-height: 100%;
+}
+
+
+.link2 {
+text-align: justify;
+color: rgb(108, 164, 140)}
+
+.woman {
+  text-align: center; 
+  line-height: 20%;
+  font-size: 30px;
+  font-family: 'Roboto Slab', serif;
+  text-decoration: none;
+  color:rgb(249, 250, 249);
+  margin: 2px;  
+}
+/*I don't want the names(links) to be underlined, 
+I want them white, why doesn't it work? */
+
+.ul {
+  list-style-type: circle;
+  text-align: center;
+  padding: 10px;
+}
+
+.infomum {
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: center;
+}
+
+.infobild {
+  width: 100%;
+}
+
+.info {
+  text-align: left;
+  font-size: 25px; 
+  font-family: 'Roboto Slab', serif ;
+  line-height: 200%;
+  color: rgb(108, 164, 140);
+  margin: 60px; 
+  height: 300px;
+  width: 300px; 
+  border-radius: 50%; 
+  padding: 50px; 
+  background-color: rgb(140, 186, 172);
+  box-shadow: 2px 2px 8px 2px  rgb(249, 250, 249); 
+}
+
+
+.flex-child {
+  color: rgb(108, 164, 140);
+  margin-left: 500px;
+  margin-bottom: 20px;  
+  padding: 10px; 
+  height: 360px;
+  width: 360px;  
+  background-color: rgb(140, 186, 172);
+  box-shadow: 2px 2px 2px 2px  rgb(249, 250, 249);  
+}
+
+.flex-brother {
+  text-align: right;
+  font-size: 11px; 
+  font-family: 'Roboto Slab', serif; 
+  margin: 0px; 
+  padding: 5px; 
+  color:rgb(21, 1, 44);
+}
+
+.flex-bild {
+  width: 40%;
+  padding: 20px; 
+}
+
+.testparent {
+  display: flex;
+  height: 700px;
+  justify-content: flex-end;
+  flex-wrap: wrap;
+}
+
+.testbox {
+  margin: 20px;
+  width: 340px;
+  height: 340px;
+  background: rgb(108, 164, 140);
+  padding: 10px;
+  background-color: rgb(140, 186, 172);
+  box-shadow: 2px 2px 2px 2px  rgb(249, 250, 249);
+}
+
+@media (min-width: 668px) and (max-width: 1024px) {
+  .sidenav {
+    width: 80%; 
+    height: auto; 
+    position: relative; 
+  }
+  .testbox {
+    width: 200px;
+    height: 350px;
+    flex-wrap: wrap;
+    flex-direction: row;
+  }
+
+  .h2 {
+    font-size: 10px;
+  }
+ 
+  }
+
+  @media (max-width: 668px) {
+    .sidenav {
+      width: 80%; 
+      height: auto; 
+      position: relative; 
+    }
+
+    .testbox {
+      flex-wrap: wrap;
+      flex-direction: row;
+      width: 300px;
+      height: 340px;
+      align-items: flex-end;
+    }
+    .h2 {
+      font-size: 10px;
+    }
+    .center {
+      width: 80%; 
+      margin-top: 100px; 
+    }
+    }
+/* I struggle to style the sidenav responsive. How can I do that?
```

Comment:
I'm not sure exactly how you want your navbar to look. One option is to remove the `position: absolute;` and to have the navbar vertical in desktop-mode, horizontal in tablet and mobile. For mobile, you could decrease the size of the links so that they fit on a small screen and add `display: flex;` to the parent so that they appear in one row. Same for tablet. For desktop, you could add `flex-direction: column;` to the parent to make them appear vertically.